pages/index.tsx

```diff
@@ -1,16 +1,19 @@
-import { SyntheticEvent, useState } from "react";
+import * as React from "react";
```

Comment:
今の React は default export されているモジュールと named export されているモジュールが一致しているので、 `import * as ~` という書き方は冗長なだけであまり意味がないという認識。
もし僕の知らない利点があるならぜひ知りたいけど、特に理由がなければ

```suggestion
import React from "react";
```
と default import にするか、 named export にしたい感がある（他のファイルも同様）

Reply:
> 今の React は default export されているモジュールと named export されているモジュールが一致しているので
これを知らなかった，それならdefault exportの体でimportしてもよさそうなので変更しておく

Reply:
と思ったら，default importもすると

Reply:
一旦named exportにして問題になってから考える